### PR TITLE
make all test/run files only symlinks to one standard file

### PR DESCRIPTION
**Pull request by zmiklank**
The run file for micro image was out-of-date and I do not see reason why it should be separated from test/run anyway.

### CODE REVIEW

**Comment by zmiklank** [comment]
[test]

**Comment by phracek** [comment]
It would be nice to have everything in one commit. It is hard to review it.

File: test/run [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+ct_init
+
+s2i_args="--pull-policy=never"
+
+run "docker inspect $IMAGE_NAME >/dev/null || docker pull $IMAGE_NAME" 0
+
+
+TEST_LIST="\
```
Comment (phracek) [inline test/run:174]: This should be at the beginning of the file, as we have it in all other containers.

File: test/run [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+run_as_root_test
+run_log_to_volume_test
+run_data_volume_test
+run_s2i_test
+run_pre_init_test
+run_mpm_config_test
+run_dockerfiles_test
+"
+
+test $# -eq 1 -a "${1-}" == --list && echo "$TEST_LIST" && exit 0
```
Comment (phracek) [inline test/run:186]: What is a reason for this? I don't follow.

**Review by phracek — CHANGES_REQUESTED** [review]
Please have a look at my comments. I would also prefer to get rid of `run` function. It is wrapper over run_command.

**Comment by zmiklank** [comment]
> It would be nice to have everything in one commit. It is hard to review it.

It is in one commit.

> Please have a look at my comments.

I did not change the contents of `test/run` file. Everything you see here was already present in the file. I only moved the file to different folder and created symlink to it. Do you want me to change it according you comments anyway?

**Comment by phracek** [comment]
CentOS Stream 9 - 2.4-micro failed for this reason:

```bash
Running transaction test
RPM: error: Unable to change root directory: Operation not permitted
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: Transaction test error:
  Errors occurred during test transaction.
```

I don't remember what was root case.

**Comment by zmiklank** [comment]
I think it can be related to change of default capabilities in podman. Fix is here: https://github.com/sclorg/container-common-scripts/pull/339.

**Comment by phracek** [comment]
The pull request https://github.com/sclorg/container-common-scripts/pull/339 is merged. Let's make changes here.

**Comment by zmiklank** [comment]
[test]

**Comment by zmiklank** [comment]
The needed changes were added.

**Pull request merged**